packages/core-data/src/utils/on-sub-key.js

```diff
@@ -12,7 +17,9 @@ export const onSubKey = ( actionProperty ) => ( reducer ) => (
 ) => {
 	// Retrieve subkey from action. Do not track if undefined; useful for cases
 	// where reducer is scoped by action shape.
-	const key = action[ actionProperty ];
+	const key = isFunction( actionProperty )
```

Comment:
Why this change? How it can be a function since you added `onSubKey( 'context' )`? In general this part confuses me the `onSubKey`..

Reply:
This is not needed anymore :) I used it at some point but I can remove it now.